backend-rust/src/graphql_api/account_metrics.sql

```diff
@@ -23,17 +23,15 @@ FROM
         $2::interval
     ) AS bucket_time
 LEFT JOIN LATERAL (
-    -- Selects the index at or before the start of the bucket.
     SELECT accounts.index
     FROM accounts
     LEFT JOIN transactions on transaction_index = transactions.index
     LEFT JOIN blocks ON transactions.block_height = height
-    WHERE slot_time <= bucket_time
```

Comment:
This could potentially mean we are operating on the same row twice providing errors

Reply:
I am not sure whether this is correct.
I think one of the bucket edges needs to include the `slot_time` otherwise a block for the exact `slot_time` would not be included in any bucket.

Reply:
So lets take an example.

We say we have two timezone 1.0 and 2.0 respectively. The pairs (with the first being the bucket timezone) can be fetched to be of the following values:

((0.9, 4), (1.9, 6)), ((2.0, 8), (\{unlimited}, 10))

This would mean we calculates 2 in the first bucket and 2 in the last. But the expected total value is 6. This is a false claim. The correct would be to use the same point as the previous after bucket, which would mean that we instead had 

((0.9, 4), (1.9, 6)), ((1.9, 6), (\{unlimited}, 10)) which gives used buckets having value 2 and 4, totalling 6 which is the correct value.

Reply:
Right, I see now that this is the correct behavior :+1: